apps/www/components/Audio/AudioPlayerController.tsx

```diff
@@ -164,12 +164,7 @@ const AudioPlayerController = ({ children }: AudioPlayerContainerProps) => {
 
   const handleError = (error: Error | string) => {
     setHasError(true)
-    if (typeof error === 'string') {
-      reportError('handle audio-error', error)
-    } else {
-      error = JSON.stringify(error, Object.getOwnPropertyNames(error), 2)
```

Comment:
This created a lot of weird-looking reports in Sentry and is completely unnecessary.